provider/github/poster.go

```diff
@@ -171,6 +171,10 @@ func (p *Poster) createReviewRequest(
 				req.Comments = append(req.Comments, comment)
 			} else {
 				line, err := dl.ConvertLine(c.File, int(c.Line))
+				if ErrLineOutOfDiff.Is(err) {
+					continue
```

Comment:
Should we log this invalid comment?

Reply:
🙏 on debug level

Reply:
Or maybe error level, if the analyzer is providing malformed comments it will be useful in production too, not only while developing.

Reply:
I didn't put log here because it's kind of expected behavior. At least for dummy analyzer. It doesn't calculate diff but scans all file. I asked ML team what they do (all file or diff).
Let's log debug/error depends on their response.

Reply:
response from ML team:
> As far as I understand changed files are processed fully (analyzer gets full uasts of previous and new version)

I'm adding it as debug level. Because it's not an error (at least for now)